analyses/efo-mondo-mapping/update_map.Rmd

```diff
@@ -0,0 +1,39 @@
+---
+title: "OLS_Search"
+output: html_document
+date: '2022-08-24'
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r load_packages, echo = FALSE}
+# load libraries
+suppressPackageStartupMessages(library(optparse))
+suppressPackageStartupMessages(library(jsonlite))
+```
+
+## R Markdown
+
+This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
+
+When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:
+
+```{Initializations}
+tab <- "\t"
+com <- ","
+
+input_file <- "efo-mondo-mapping/results/efo-mondo-map-prefill.tsv"
+cg_data <- readr::read_tsv(file = input_file,col_names = TRUE)
```

Comment:
better to use file.path() for this

Reply:
Removed the file since it is not needed.